test/Prototypes/DoubleWidth.swift.gyb

```diff
@@ -282,15 +282,15 @@ extension DoubleWidth.Words: Collection {
 
   public func index(after i: Index) -> Index {
     switch i._value {
+    case let .low(li) where Base.bitWidth < UInt.bitWidth:
+      return Index(.high(_high.endIndex)) 
     case let .low(li):
```

Comment:
Hoping the optimizer makes this just as good, cos I find it much more readable...